alfresco-transform-core-aio/alfresco-transform-core-aio-boot/src/main/java/org/alfresco/transformer/AIOController.java

```diff
@@ -118,9 +118,9 @@ protected void executeTransformCommand(File sourceFile, File targetFile)
     @PostMapping(value = "/transform", consumes = MULTIPART_FORM_DATA_VALUE)
     public ResponseEntity<Resource> transform(HttpServletRequest request,
         @RequestParam("file") MultipartFile sourceMultipartFile,
+        @RequestParam(TARGET_EXTENSION) String targetExtension,

```

Comment:
Existing tests expect this parameter to be checked first.

Reply:
It's checked as a negative test. This is the minimum parameter required across all T-engines and the Super test class AbstractTransformerControllerTest only checks for an error on the TARGET_EXTENSION parameter.  If any other parameters are run before (and not included) it does not present the expected errror.

Reply:
Ok, thanks. 

As an aside, I'm not sure this should be the minimum param now for a well-defined T-API.